telegram/_telegramobject.py

```diff
@@ -169,28 +170,28 @@ def _get_attrs(
         """
         data = {}
 
-        if not recursive:
-            try:
-                # __dict__ has attrs from superclasses, so no need to put in the for loop below
-                data.update(self.__dict__)
-            except AttributeError:
-                pass
+        # __dict__ has attrs from superclasses, so no need to recompute/duplicate
```

Comment:
what exactly do you mean by "recompute/duplicate" here? If the parent has `__slots__`, then those aren't included in `__dict__` …

Reply:
hmm I guess I can revert that comment